vpr/src/pack/pack.cpp

```diff
@@ -184,12 +184,22 @@ bool try_pack(t_packer_opts* packer_opts,
             attraction_groups.set_att_group_pulls(1);
 
         } else if (pack_iteration >= 2 && pack_iteration < 5 && floorplan_not_fitting) {
-            VTR_LOG("Floorplan regions are overfull: trying to pack again with more attraction groups exploration and higher target pin utilization. \n");
-            attraction_groups.create_att_groups_for_overfull_regions();
-            VTR_LOG("Pack iteration is %d\n", pack_iteration);
-            attraction_groups.set_att_group_pulls(4);
-            t_ext_pin_util pin_util(1.0, 1.0);
-            target_external_pin_util.set_block_pin_util("clb", pin_util);
+            if (pack_iteration == 2) {
```

Comment:
add comments about what our strategy is here: what do we do differently in each iteration

Reply:
also might just want to make a loop with a defined number of max packing iterations